.vscode/cspell.json

```diff
@@ -504,6 +504,29 @@
         "undoc"
       ]
     },
+    {
+      "filename": "sdk/storage/azure-storage-file-datalake/**",
+      "words": ["myfile", "dlake"]
+    },
+    {
+      "filename": "sdk/storage/azure-storage-file-datalake/**/*.py",
+      "words": [
+        "rwxrwxrwx", "rwxrw", "rwxr", "ABFS", "deletionid", "saoid", "testfs",
+        "mynewfilesystems", "myleasefilesystem", "mymetadatafilesystemsync",
+        "mypathfilesystem", "myclientfilesystem", "mydirectoryfilesystem", "myfilesystemforgetclient",
+        "mynewdirectory", "mydirectory", "mynewfile", "myfilesystem", "mynewfilesystem",
+        "myfilesystemforlistpaths", "cpkfile", "dont", "fsname", "fpath", "dname", "racwdlmeop",
+        "filesys", "nonexistentfs", "testid", "cpkdirectory", "cpksubdirectory"
+      ]
+    },
+    {
+      "filename": "sdk/storage/azure-storage-file-datalake/**/*.md",
+      "words": ["thead", "tbody", "myfilesystem"]
+    },
```

Comment:
Could you combine all the datalake ones into one section `"sdk/storage/azure-storage-file-datalake/**"`.

Can you also do one word per line to match the style of the rest of the files.

Reply:
Hello @jalauzon-msft ! I have reordered the words into one section as you requested. Thank you so much for your work :)

Reply:
Thanks! Looks good.